Build failure: lxqt.lxqt-config

### Nixpkgs version

- Unstable (26.05)

### Steps to reproduce

Build *lxqt.lxqt-config* on unstable https://github.com/NixOS/nixpkgs/commit/0182a361324364ae3f436a63005877674cf45efb

### Can Hydra reproduce this build failure?

Yes, Hydra can reproduce this build failure.

### Link to Hydra build job

https://hydra.nixos.org/build/321960832

### Relevant log output

```console
error: Cannot build '/nix/store/dnsr4b5falagc2igfkjxxbm0bm62h2zd-lxqt-config-2.3.0.drv'.
       Reason: builder failed with exit code 2.
       Output paths:
         /nix/store/vycskmp1p70wyfwapidf835sda5mi2c7-lxqt-config-2.3.0
       Last 25 log lines:
       >     Generated 1 translation(s) (1 finished and 0 unfinished)
       > Detected locale "C" with character encoding "ANSI_X3.4-1968", which is not UTF-8.
       > Qt depends on a UTF-8 locale, and has switched to "C.UTF-8" instead.
       > If this causes problems, reconfigure your locale. See the locale(1) manual
       > for more information.
       > Updating '/build/source/build/src/lxqt-config_vi.qm'...
       >     Generated 1 translation(s) (1 finished and 0 unfinished)
       > Detected locale "C" with character encoding "ANSI_X3.4-1968", which is not UTF-8.
       > Qt depends on a UTF-8 locale, and has switched to "C.UTF-8" instead.
       > If this causes problems, reconfigure your locale. See the locale(1) manual
       > for more information.
       > Updating '/build/source/build/src/lxqt-config_uk.qm'...
       >     Generated 1 translation(s) (1 finished and 0 unfinished)
       > [ 69%] Building CXX object src/CMakeFiles/lxqt-config.dir/mainwindow.cpp.o
       > [ 69%] Building CXX object src/CMakeFiles/lxqt-config.dir/lxqt-config_autogen/mocs_compilation.cpp.o
       > [ 70%] Building CXX object src/CMakeFiles/lxqt-config.dir/main.cpp.o
       > [ 70%] Building CXX object src/CMakeFiles/lxqt-config.dir/LXQtAppTranslationLoader.cpp.o
       > make[1]: *** [CMakeFiles/Makefile2:893: lxqt-config-monitor/CMakeFiles/lxqt-config-monitor.dir/all] Error 2
       > make[1]: *** Waiting for unfinished jobs....
       > [ 70%] Built target lxqt-config-file-associations
       > [ 71%] Linking CXX shared library liblxqt-config-cursor.so
       > [ 71%] Linking CXX executable lxqt-config
       > [ 72%] Built target lxqt-config-cursor
       > [ 72%] Built target lxqt-config
       > make: *** [Makefile:136: all] Error 2
       For full logs, run:
         nix log /nix/store/dnsr4b5falagc2igfkjxxbm0bm62h2zd-lxqt-config-2.3.0.drv
```

### Additional context

_No response_

### System metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 6.19.2, NixOS, 26.05 (Yarara), 26.05.20260217.0182a36`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.33.3`
 - channels(root): `""`
 - nixpkgs: `/nix/store/dx2qikyb4dyb6hbdfywbmsyla0z5a1h3-source`

### Notify maintainers

@romildo 

---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+label%3A%220.kind%3A+build+failure%22).
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/desktops/lxqt/lxqt-config/default.nix
+++ b/pkgs/desktops/lxqt/lxqt-config/default.nix
@@ -26,15 +26,15 @@
   gitUpdater,
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "lxqt-config";
-  version = "2.3.0";
+  version = "2.3.1";
 
   src = fetchFromGitHub {
     owner = "lxqt";
     repo = "lxqt-config";
-    rev = version;
-    hash = "sha256-2CAQeX2X0DPmgOaAEJoCLtgjFT+Z6epc/dUCbaEIlB0=";
+    tag = finalAttrs.version;
+    hash = "sha256-2fviPhSBwUU9jg3217PLbREh8MkArd2Uc4bhFXo2J7U=";
   };
 
   nativeBuildInputs = [
@@ -86,4 +86,4 @@ stdenv.mkDerivation rec {
     teams = [ lib.teams.lxqt ];
   };
 
-}
+})

